cmake/gen_cmake_skeleton.py

```diff
@@ -278,11 +292,10 @@ def get_test_exe_name(filename):
         ret.append("""
 install(TARGETS {tgt}
     EXPORT kaldi-targets
-    ARCHIVE DESTINATION ${{CMAKE_INSTALL_LIBDIR}}
-    LIBRARY DESTINATION ${{CMAKE_INSTALL_LIBDIR}}
-    RUNTIME DESTINATION ${{CMAKE_INSTALL_BINDIR}}
+    ARCHIVE DESTINATION ${{CMAKE_INSTALL_LIBDIR}} COMPONENT kaldi
+    LIBRARY DESTINATION ${{CMAKE_INSTALL_LIBDIR}} COMPONENT kaldi
+    RUNTIME DESTINATION ${{CMAKE_INSTALL_BINDIR}} COMPONENT kaldi
```

Comment:
Ditto w.r.t. `DESTINATION` and `COMPONENT`. This reduces to either
```
install(TARGETS {tgt} EXPORT kaldi-targets)
```
or
```
install(TARGETS {tgt} EXPORT kaldi-targets COMPONENT kaldi)
```
depending on the requrement of the Conda CMake bild to provide explicit component name(s).